test/e2e/framework/helpers/common/utils.go

```diff
@@ -116,7 +114,6 @@ func GetUUID() string {
 
 func RandSeq(n int) string {
 	var letters = []rune("abcdefghijklmnopqrstuvwxyz0123456789")
-	rand.Seed(time.Now().UnixNano())
```

Comment:
Not sure why this one is removed. The seeding can stay as far I can tell.

Reply:
This is no longer needed in Go 1.20, same discussion I had with Craig on the placeholder name generation. Yes we can remove the Seed() call.